Games/Retro_ML.SuperMarioKart/Neural/Scoring/LakituScoreFactor.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Retro_ML.Game;
```

Comment:
Modify lines 91-96 so that they no longer use the ExtraField

https://github.com/Tourmi/Retro-ML/pull/27/files#diff-2bbcf07e573e2c9f1636f0df8ec6313db772736f33b2bc1dc712b9edc336e3b5R91-R96